apps/financial-aid/api/src/app/modules/application/dto/updateApplication.input.ts

```diff
@@ -24,6 +24,10 @@ export class UpdateApplicationInput implements UpdateApplication {
   @Field(() => String)
   readonly event!: ApplicationEventType
 
+  @Allow()
+  @Field({ nullable: true })
+  readonly applied?: string
```

Comment:
I know I suggested this name, but I kind off feel like when I read applied that it is a boolean as in has this party applied. 

Wondering if we should change it to appliedMonth, appliedPeriod, appliedDate or something like that?

Reply:
appliedDate sounds good!